wordpress-mobile/WordPress-iOS
# Transitioned People view to use the Filter bar

### Description
Fixes #9779, which called for the `FilterTabBar` to be integrated in new places, including the _People_ view.

### Testing

1. Checkout the branch and verify that existing tests pass.
1. Navigate to _My Sites : Site : People_ and confirm that the appropriate filter tab bar is presented. It might be worthwhile to review in the Simulator to confirm that no constraint warnings are presented. (Note that _Viewers_ should only be visible on private sites.)
1. Verify that the results update when you select different choices, and that the _No Results_ view is appropriately centered when presented.

### Screenshots

_Before_

<img width="359" alt="9779-before" src="https://user-images.githubusercontent.com/221062/48217645-c9eb8b00-e33c-11e8-81e9-cfcdd1740ac7.png">

_After_

![9779-after](https://user-images.githubusercontent.com/221062/48217616-b5a78e00-e33c-11e8-9719-a41973b86951.png)

![9779-after-noresults](https://user-images.githubusercontent.com/221062/48217619-bb04d880-e33c-11e8-8880-d69209c6e472.png)
